build/teamcity/cockroach/create-docs-issue.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# This script is called by the build configuration
+# "Cockroach > Create Docs Issue" in TeamCity.
+
+set -euxo pipefail
+
+dir="$(dirname $(dirname $(dirname "${0}")))"
+source "$dir/teamcity-support.sh"
+source "$dir/teamcity-bazel-support.sh"
+
+BAZEL_SUPPORT_EXTRA_DOCKER_ARGS="-e GITHUB_API_TOKEN -e BUILD_VCS_NUMBER" run_bazel << 'EOF'
+bazel build --config ci //pkg/cmd/docs-issue-generation
+BAZEL_BIN=$(bazel info bazel-bin --config ci)
+$BAZEL_BIN/pkg/cmd/docs-issue-generation/docs-issue-generation_/docs-issue-generation
```

Comment:
We don't really use Bazel for these old branches and while this may work for 21.2, it's kind of odd. Meanwhile I have little faith this will work for 21.1 either.

I think for these older branches it may work to do something like this:

```
build/builder.sh go build -o bin/docs-issue-generation ./pkg/cmd/docs-issue-generation
./bin.docker_amd64/docs-issue-generation
```

Reply:
Done.